tasks/sep/base-005-fp-holocene-upgrade/README.md

```diff
@@ -0,0 +1,35 @@
+# Holocene Hardfork Upgrade - Base Sepolia
+
+Status: READY TO SIGN
+
+## Objective
+
+Upgrades the Fault Proof contracts of Base Sepolia for the Holocene hardfork.
+
+This upgrades the Fault Proof contracts in the
+[op-contracts/v1.8.0-rc.1](https://github.com/ethereum-optimism/optimism/tree/op-contracts/v1.8.0-rc.1) release.
```

Comment:
It may be important that the rc.2 release was used for the deployments of MIPS, FDG, PDG - @clabby @mds1 ?

Reply:
From discord it appears the rc.2 tag was used? From https://discord.com/channels/1244729134312198194/1309378748331855885/1310587708720156764:
> Sorry forgot to update this - the contracts were deployed using rc.2 (you can see in the dispute game implementation that the `beta` tag isn't there which is only an rc.2 change I believe)